patterns/appmesh-mtls/README.md

```diff
@@ -17,11 +17,11 @@ See [here](https://aws-ia.github.io/terraform-aws-eks-blueprints/getting-started
     ```text
     NAMESPACE          NAME                                       READY   STATUS    RESTARTS   AGE
     appmesh-system     appmesh-controller-7c98b87bdc-q6226        1/1     Running   0          4h44m
-    cert-manager       cert-manager-87f5555f-tcxj7                1/1     Running   0          4h43m
-    cert-manager       cert-manager-cainjector-8448ff8ddb-wwjsc   1/1     Running   0          4h43m
-    cert-manager       cert-manager-webhook-5468b675b-fvdwk       1/1     Running   0          4h43m
+    cert-manager       cert-manager-87f5555f-t1xj7                1/1     Running   0          4h43m
```

Comment:
the spell checker isn't super great - if it sees 4 or more characters in a row it will treat it as a word. so something like `cert-manager-87f5555f-tcxj7 ` will fail spell check but if you break it up with a numeric value like `cert-manager-87f5555f-t1xj7` it ignores it 🤷🏽